packages/feedback-react/src/FeedbackValues.tsx

```diff
@@ -1,5 +1,7 @@
-export const VERY_UNHAPPY = 1;
-export const UNHAPPY = 2;
-export const NEUTRAL = 3;
-export const HAPPY = 4;
-export const VERY_HAPPY = 5;
+import { FeedbackValue } from "./types";
+
+export const VERY_UNHAPPY: FeedbackValue = 1;
```

Comment:
Dette er bare for å gjøre det litt enklere å bruke disse verdiene i klient. Slik det var ble de typet som `number` ved import.